virtusize/src/main/java/com/virtusize/android/util/VirtusizeUtils.kt

```diff
@@ -159,24 +157,6 @@ internal object VirtusizeUtils {
     /**
      * A function to open the Virtusize WebView
      */
-    fun openVirtusizeWebView(
-        context: Context,
-        virtusizeParams: VirtusizeParams?,
-        virtusizeDialogFragment: VirtusizeWebViewFragment,
-        product: VirtusizeProduct,
-        messageHandler: VirtusizeMessageHandler,
-    ) {
-        val fragmentTransaction = (context as FragmentActivity).supportFragmentManager.beginTransaction()
-        val previousFragment = context.supportFragmentManager.findFragmentByTag(Constants.FRAG_TAG)
-        previousFragment?.let { fragment ->
-            fragmentTransaction.remove(fragment)
-        }
-        fragmentTransaction.addToBackStack(null)
-        virtusizeDialogFragment.arguments = createBundle(virtusizeParams, product)
-        virtusizeDialogFragment.setupMessageHandler(messageHandler)
-        virtusizeDialogFragment.show(fragmentTransaction, Constants.FRAG_TAG)
-    }
-
```

Comment:
This is deleted as it will crash the app if the clients don't use the Activity that inherits FragmentActivity for their app, for the apps that use Jetpack Compose, Fragments are rarely used.

Reply:
OK.